src/app/components/content-page/content-page.component.ts

```diff
@@ -1,50 +1,73 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
 import { Router } from '@angular/router';
+import { AngularFireDatabase } from '@angular/fire/database';
+import { AngularFireAuth } from '@angular/fire/auth';
+
 import { faPlusSquare } from '@fortawesome/free-solid-svg-icons';
 import { faSearch } from '@fortawesome/free-solid-svg-icons';
 
-import { AngularFireDatabase } from '@angular/fire/database';
-import { AngularFireAuth } from '@angular/fire/auth';
+import { ToastrService } from 'ngx-toastr';
 
 import { StorageService } from 'src/app/services/storage.service';
 import { Image } from 'src/app/models/image';
 import { User } from 'src/app/models/user';
+import { Subscription } from 'rxjs';
 
 @Component({
   selector: 'app-content-page',
   templateUrl: './content-page.component.html',
   styleUrls: ['./content-page.component.scss']
 })
 
-export class ContentPageComponent implements OnInit {
+export class ContentPageComponent implements OnInit, OnDestroy {
   faPlusSquare = faPlusSquare;
   faSearch = faSearch;
   isSearchCollapsed = true;
 
   user: User;
-  firebaseStorage: any;
   images: Image[];
+  uploadSubsribtion: Subscription;
+  imageSubsribtion: Subscription;
 
   constructor(private firebaseAuth: AngularFireAuth,
               public db: AngularFireDatabase,
               private storage: StorageService,
-              private router: Router) {}
+              private router: Router,
+              private toastr: ToastrService) { }
 
   ngOnInit() {
     this.firebaseAuth.auth.onAuthStateChanged(user => {
-    if (!user) {
-      this.router.navigateByUrl('/login');
-    } else {
-      this.firebaseStorage = this.db.list<Image>(user.uid + '/files')
-      .valueChanges()
-      .subscribe(values => {
-          this.images = values;
-      });
-    }
-  });
-}
+      if (!user) {
+        this.router.navigateByUrl('/login');
+      } else {
+        this.db.list<Image>(user.uid + '/files')
+          .valueChanges()
+          .subscribe(values => {
+              this.images = values;
+          });
+      }
+    });
+    this.uploadSubsribtion = this.storage.isUploadedSource.subscribe(
+      isUploaded => {
+        if (isUploaded) {
+          this.displayToast();
+        }
+      }
+    );
+  }
+
+  ngOnDestroy() {
+    this.uploadSubsribtion.unsubscribe();
+  }
 
   uploadFile($event: any): void {
     this.storage.uploadFile($event.target.files[0]);
   }
+
+  displayToast() {
+    this.toastr.success('upload complete', 'Success!', {
+      closeButton: true,
+      progressBar: true
```

Comment:
i think it might look a little cleaner if we remove the progress bar

Reply:
also is it possible to move the placement of the toast? at the moment its over the nav bar buttons

Reply:
I agree with progress bar removal.

Reply:
Where would you like to move the toast?

Reply:
maybe just a little down. is that possible?

Reply:
According to what I understand I can only do the following in terms o f positioning:
Top Right
Bottom Right
Bottom Left
Top Left
Top Full Width
Bottom Full Width
Top Center
Bottom Center

Reply:
it might be possible with this 
https://www.npmjs.com/package/ngx-toastr#put-toasts-in-your-own-container 

but it could be difficult seeing as were also changing the app structure.

Reply:
Okay what about bottom-right?

Reply:
you can leave it for now, bottom right will interfere with mobile view

Reply:
Cool man! 👍